mailpoet/tests/integration/Cron/Workers/SubscribersLifetimeEmailCountTest.php

```diff
@@ -0,0 +1,178 @@
+<?php declare(strict_types=1);
+
+namespace MailPoet\Cron\Workers;
+
+use MailPoet\Entities\NewsletterEntity;
+use MailPoet\Entities\ScheduledTaskEntity;
+use MailPoet\Entities\ScheduledTaskSubscriberEntity;
+use MailPoet\Entities\SendingQueueEntity;
+use MailPoet\Entities\SubscriberEntity;
+use MailPoet\Models\ScheduledTask;
+use MailPoet\Newsletter\Sending\ScheduledTasksRepository;
+use MailPoet\Subscribers\SubscribersRepository;
+use MailPoet\Tasks\Sending;
+use MailPoet\Test\DataFactories\ScheduledTask as ScheduledTaskFactory;
+use MailPoet\WP\Functions as WPFunctions;
+use MailPoetVendor\Carbon\Carbon;
+
+class SubscribersLifetimeEmailCountTest extends \MailPoetTest {
```

Comment:
I had to rename this file so that the test is executed after `SubscribersLastEngagementTest.php`
I haven't been able to figure out what is the side effect but if it runs before, the test `MailPoet\Cron\Workers\SubscribersLastEngagementTest::testItCanSetLastEngagementFromOpens` fails.

Reply:
Do you think we should investigate this further? (E.g. create a ticket?)

Reply:
Yes, I will create the ticket.